package.json

```diff
@@ -11,16 +11,8 @@
     "lint": "eslint --ext '.jsx,.js' ./",
     "build-babel": "babel -D --ignore '__tests__,__mocks__' packages/$PACKAGE/src -d packages/$PACKAGE/lib",
     "build-webpack": "webpack --config packages/$PACKAGE/webpack.config.js",
-    "build:playground": "PACKAGE=react-component-playground npm run build-webpack",
-    "build:component-tree": "PACKAGE=react-component-tree npm run build-babel",
-    "build:cosmos": "PACKAGE=react-cosmos npm run build-babel",
-    "build:utils": "PACKAGE=react-cosmos-utils npm run build-babel",
-    "build:webpack": "PACKAGE=react-cosmos-webpack npm run build-babel",
-    "build:dom-polyfill": "PACKAGE=react-dom-polyfill npm run build-babel",
-    "build:querystring-router": "PACKAGE=react-querystring-router npm run build-babel",
-    "build:redux-proxy": "PACKAGE=react-cosmos-redux-proxy npm run build-babel",
-    "build:context-proxy": "PACKAGE=react-cosmos-context-proxy npm run build-babel",
-    "build": "rm -rf ./packages/*/lib && npm run build:utils && npm run build:playground && npm run build:component-tree && npm run build:cosmos && npm run build:webpack && npm run build:dom-polyfill && npm run build:querystring-router && npm run build:redux-proxy && npm run build:context-proxy",
+    "build:package": "node ./scripts/build-package.js",
+    "build": "rm -rf ./packages/*/lib && npm run build:package",
```

Comment:
Having these two scripts is confusing, it took me a bit to understand what happens. E.g.

```bash
# Both build only the react-cosmos-utils script, except in the first case the watch flag is ignored
npm run build react-cosmos-utils -- --watch
npm run build:package react-cosmos-utils -- --watch
```

Let's get rid of `build:package`. You can use rimraf (see [example](https://github.com/react-cosmos/react-cosmos/blob/abb3dedfaa07e72a6c2b95a9e898d11d579579a2/scripts/prune-react-copies.js#L9-L11)) to run wipe out `./packages/*/lib` when `targetPackage` is missing.